lib/routes/account.js

```diff
@@ -491,6 +491,11 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
                   // deleted the original email they signed up with
                   return db.emailRecord(email)
                     .then((emailRecord) => {
+                      if (originalLoginEmail === undefined) {
+                        // User deleted original email and attempted to login with it,
+                        throw error.cannotLoginWithSecondaryEmail()
```

Comment:
Is there a mismatch between the comment and the error here? Maybe not as I don't have all the context, but the comment seems to imply they tried to log in with their original (deleted) email and the error seems to imply they tried to log in with a secondary email.

Not a big deal obviously, just wanted to check whether it warrents a change to the comment or a new error type?